test/commands/distribute/groups/update-test.ts

```diff
@@ -67,12 +126,14 @@ describe("distribute groups update command", () => {
   function testCommandSuccess(result: CommandResult, executionScope: Nock.Scope, abortScope?: Nock.Scope) {
     console.log(result);
     expect(result.succeeded).to.eql(true, "Command should be successfully completed");
-    expect(abortScope.isDone()).to.eql(false, "Unexpected requests were made");
+    if (abortScope) {
+      expect(abortScope.isDone()).to.eql(false, "Unexpected requests were made");
+    }
     executionScope.done(); // All normal API calls are executed
   }
 
-  function testCommandFailure(executionScope: Nock.Scope, abortScope?: Nock.Scope) {
-    expect(abortScope.isDone()).to.eql(false, "Unexpected requests were made");
+  function testCommandFailure(executionScope: Nock.Scope, skippedScope?: Nock.Scope) {
+    expect(skippedScope.isDone()).to.eql(false, "Unexpected requests were made");
```

Comment:
Why doesn't it have null check like `testCommandSuccess`?

Reply:
A better question would be why is this method copied into every test class 😛 
I didn't need to fix this one to make my code work.